participants/qareports.py

```diff
@@ -1,10 +1,4 @@
-import os
-
-
-from qarep.upload import (
-    ReportUploader, get_results_files_list, move_results_dir
-)
-
+from qarep.upload import *
```

Comment:
Hmm, I consider the qareports in dister code to be authorative.  More checking required.